src/Pages/byulyi/Login/Login.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react'
+import './Login.scss'
+
+class Login extends Component {
+    state = {
+      user : {
+        id: '',
+        password: ''
+      }
+    }
+
+  handleIdChange = (e) => {
+    const user = {...this.state.user, id:e.target.value};
+    this.setState({user});
+  }
+
+  handlePwChange = (e) => {
+    const user = {...this.state.user, password:e.target.value};
+    this.setState({user});
+  }
+
+  handleSubmitBtn = (e) => {
```

Comment:
사용지 않는 매개변수는 삭제해 주세요!
```suggestion
  handleSubmitBtn = () => {
```